server/sonar-web/src/main/js/apps/projects/components/AllProjects.js

```diff
@@ -72,6 +75,12 @@ export default class AllProjects extends React.PureComponent {
 
   isFiltered = () => Object.keys(this.state.query).some(key => this.state.query[key] != null);
 
+  getSavedOptions = () => ({
+    sort: utils.getSort(),
+    view: utils.getView(),
+    visualization: utils.getVisualization()
```

Comment:
We end up with some parameters set to nothing if we don't have values in the localstorage, maybe they should also not be set if the default values are selected like "overall", "name" or "risk".